docs/Additional Information/contract-devs-faq.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: "Contract Developers FAQ"
+permalink: "docs/contract-developers-faq/"
+metadata: 
+  title: "Contract Developer FAQ"
+  description: "Find answers to common questions about developing on Chainlink."
+---
+
+## General FAQ
```

Comment:
Can we take the information in this FAQ and place it in the related documents in non-FAQ format? I think many of these questions have associated documentation where they belong.

Reply:
Not sure what other document do you want this to go to. This is a fairly regular question people have been asking for and still continue to ask for some reason.

Reply:
I think I know where I want to reorganize this information. Give me some time to go through it and I'll put together a draft.